src/main/kotlin/step3/carRacing/contoller/RaceController.kt

```diff
@@ -0,0 +1,22 @@
+package step3.carRacing.contoller
+
+import step3.carRacing.model.Car
+import step3.carRacing.view.OutputView
+import step3.carRacing.view.SystemPrint
+import kotlin.random.Random
+
+class RaceController {
+    private fun tryRound(cars: List<Car>) {
+        for (car in cars) {
+            car.tryRace(Random.nextInt(0, 10))
```

Comment:
for 문을 활용해볼 수도 있지만, Collections 확장 함수인 `forEach`를 활용해보는 건 어떨까요?
```suggestion
        cars.forEach { car -> ... }
```